source/puppet/4.10/lang_data_number.md

```diff
@@ -109,28 +122,27 @@ The full signature for `Integer` is:
 
 All of these parameters are optional. They must be listed in order; if you need to specify a later parameter, you must specify values for any prior ones.
 
-Position | Parameter        | Data Type | Default Value | Description
----------| -----------------|-----------|---------------|------------
-1 | Min value | `Integer` | negative infinity | The minimum value for the integer. This parameter accepts the special value `default`, which will use its default value.
-2 | Max value | `Integer` | infinity | The maximum value for the integer. This parameter accepts the special value `default`, which will use its default value.
+Position | Parameter        | Data type | Default value     | Description
+---------| -----------------|-----------|-------------------|-----------------------------
+1        | Minimum value    | `Integer` | negative infinity | The minimum value for the integer. This parameter accepts the special value `default`, which uses its default value.
+2        | Maximum value    | `Integer` | infinity          | The maximum value for the integer. This parameter accepts the special value `default`, which uses its default value.
```

Comment:
it isn't really infinity and negative infinity, it is MAXINT, and MININT the respective max/min a 64 bit integer can describe. (The values shown below).